### PR TITLE
Add everything from `regionprops`

### LINKED ISSUE #90: regionprops
- cell elongation or min/max axes
- neighbours indexes
- convex shape or circularity
- polygon class
- c1 translations (requires tracking)

### PR DESCRIPTION
**Pull request by paddyroddy**
Fixes #90.

Researchers mentioned that it would be useful to have everything from `regionprops` rather than having only certain things.

There was some discussion with units in #52. However, the researchers thought it would be easier to just use the `regionprops` defaults.

See #96, #97, #98 & #100 for more detail on some of the changes.

Happy to discuss. This was more complicated than I thought it would be.

### CODE REVIEW

File: src/epitools/analysis/cell_statistics.py [hunk 1]
```diff
@@ -113,8 +112,6 @@ def _calculate_cell_statistics(
     # skimage uses 'label' for what napari calls 'index'
     for frame_stats in cell_statistics:
         frame_stats["index"] = frame_stats.pop("label")
-        frame_stats["perimeter"] *= 1e6  # convert to um
-        frame_stats["area"] *= 1e12  # convert to um2
```
Comment (paddyroddy) [inline src/epitools/analysis/cell_statistics.py:117]: Removing unit conversions

File: src/epitools/widgets/cell_statistics.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+import skimage.measure._regionprops
```
Comment (paddyroddy) [inline src/epitools/widgets/cell_statistics.py:3]: Same problem

File: tests/test_cell_statistics.py [hunk 3]
```diff
@@ -68,20 +68,20 @@ def test_segmentation_widget_run_button(
     )
 
 
-def test_calculate_cell_statistics(
```
Comment (paddyroddy) [inline tests/test_cell_statistics.py:71]: This test loads a pre-calculated data file so think will fail when we change stuff

File: src/epitools/widgets/cell_statistics.py [hunk 4]
```diff
@@ -88,7 +88,7 @@ def _create_colour_labels_widgets() -> list[Widget]:
         label="statistic",
         widget_type="ComboBox",
         options={
-            "choices": ["id", "area", "perimeter", "orientation", "neighbours"],
+            "choices": ["id", "orientation", "neighbours"],
```
Comment (paddyroddy) [inline src/epitools/widgets/cell_statistics.py:91]: These only work with the unit conversions, but I would need to work them out for all other `regionprops` so just removing the colourmaps for now.

File: src/epitools/analysis/cell_statistics.py [hunk 5]
```diff
@@ -20,6 +20,7 @@
 import numpy as np
 import skimage.graph
 import skimage.measure
+import skimage.measure._regionprops
```
[the remark this answers is not in the document]
Comment (paddyroddy) [reply src/epitools/analysis/cell_statistics.py]: Not relevant any more

File: src/epitools/widgets/cell_statistics.py (hunk 2 above)
Comment (paddyroddy) [reply src/epitools/widgets/cell_statistics.py]: Not relevant any more

**Review by alessandrofelder — APPROVED** [review]
OK - well done on reaching a wrapping-up point with next steps for future work in issues.

**Pull request merged**